### PR TITLE
David emails

### PR DESCRIPTION
**Pull request by dkoh12**

### CODE REVIEW

File: app/mailers/admin_mailer.rb [hunk 1]
```diff
@@ -4,11 +4,35 @@ class AdminMailer < ApplicationMailer
...
       @admin = admin
       @url  = 'http://www.gmail.com'
-      mail(to: @admin.email, subject: 'Welcome to EnrollMe') do |format|
+      mail(to: @admin.email, subject: 'Here is list of approved teams in EnrollMe') do |format|
         format.html
       end
    end
    
+   def send_approved_email(team)
+      @team = team
```
Comment (jjlgao) [inline app/mailers/admin_mailer.rb]: We should have an assertion to make sure that the team is approved/disapproved before emailing them.

File: app/mailers/admin_mailer.rb [hunk 2]
```diff
@@ -4,11 +4,35 @@ class AdminMailer < ApplicationMailer
...
+      
+      @team.users.each do |user|
+         @user = user
+         @url = 'https://cs-169-dasolyoon.c9users.io'
```
Comment (jjlgao) [inline app/mailers/admin_mailer.rb]: Is this correct? You should grab the URL from somewhere dynamic by adding it into the environment or having Heroku do it for you. Shouldn't be hardcoded like this.

File: app/mailers/email_students.rb [hunk 3]
```diff
@@ -11,5 +11,16 @@ def submit_email(user)
        mail(to: @user.email, subject: 'Don\'t forget to submit team for CS 169')
     end
     
+    def successfully_submitted_email(team)
+      @team = team
```
Comment (jjlgao) [inline app/mailers/email_students.rb]: Should we have an assertion here as well? Would make sure you can't accidentally send these emails without also approving/disapproving/submitting these teams first.

File: app/mailers/email_students.rb [hunk 4]
```diff
@@ -11,5 +11,16 @@ def submit_email(user)
...
+      
+      @team.users.each do |user|
+         @user = user
+         @url = 'https://cs-169-dasolyoon.c9users.io'
```
Comment (jjlgao) [inline app/mailers/email_students.rb]: URL again

File: config/environments/development.rb [hunk 5]
```diff
@@ -60,7 +60,7 @@
     port: 587,
     domain: 'gmail.com',
     user_name: 'enrollmeberkeley@gmail.com',
-    password: 'eecseecseecs',
+    password: 'xsuexspmvwbyifht',
```
Comment (jjlgao) [inline config/environments/development.rb]: Should this be left to Adnan's other PR?

File: app/mailers/email_students.rb (hunk 3 above)
Comment (dkoh12) [inline app/mailers/email_students.rb]: I feel like the assertions should go in the controller. not inside the mailer

**Pull request merged**